quickjs.c

```diff
@@ -1843,9 +1843,13 @@ JSRuntime *JS_NewRuntime2(const JSMallocFunctions *mf, void *opaque)
     rt->js_class_id_alloc = JS_CLASS_INIT_COUNT;
 
     rt->stack_size = JS_DEFAULT_STACK_SIZE;
-#ifdef __ASAN__
-    rt->stack_size *= 2; // stack frames are bigger under AddressSanitizer
+//#ifdef __ASAN__
+//    rt->stack_size *= 2; // stack frames are bigger under AddressSanitizer
+//#endif
```

Comment:
This can go then, it looks like?
```suggestion
```

Reply:
Yep, I just wanted to check if tests passed.